docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,85 @@
+# Broadcast Rules For Elementwise Operations {#openvino_docs_ops_broadcast_rules}
+
+The purpose of this document is to provide a set of common rules which are applicable for ops using broadcasting.
+
+## Description
+
+Broadcast allows to perform element-wise operation for inputs with different shapes. Smaller input is *stretched* into an array with the same shape as the larger input. As a result both inputs have compatible shapes.
```

Comment:
The sentence "Smaller input is *stretched* into an array with the same shape as the larger input." is not applicable anymore.

Reply:
Sentence removed.